app/assets/stylesheets/application_layout.scss

```diff
@@ -9,7 +9,7 @@ body {
   background-color: $application_background_color;
   color:            $os_gray;
   margin-top:       0px;
-  background: image_url('home-bg.jpg') no-repeat center center fixed;
+  background: image_url('bg-login.jpg') no-repeat center center fixed;
   background-size: cover;
```

Comment:
`grep -r home-bg.jpg app/assets/stylesheets/*` suggests that's the only spot it's referenced